### ISSUE TITLE
Translate Manuscript Viewer to German

### ISSUE THREAD

**Issue by uliska** [wbsoft/frescobaldi#719]

**Comment by uliska** [comment]
I think I will defer that to when we have merged the viewer (to avoid an - improbable - clash with someone else working on the German translation

**Comment by fedelibre** [comment]
now that it's merged, should we close this issue?

**Comment by uliska** [comment]
Not really, as this issue doesn't indicate I did it but that I'd like to do that.

The problem is that I *still* can't run anything newer than 2.19. due to that strange Python memory access error.
